circle.yml

```diff
@@ -23,13 +23,48 @@ executors:
       xcode: 11.6.0
 
 commands:
-  description: "Install macOS system dependencies"
   install_macos_deps:
+    description: "Install macOS system dependencies"
```

Comment:
What is the difference?

Reply:
It was in wrong place. This is `description` of `install_macos_deps` command.